### ISSUE TITLE
[Feature Request]: Blending in Stable Diffusion

### ISSUE THREAD

**Issue by neverix** [deforum-art/sd-webui-text2video#149]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Modelscope and SD use the same autoencoder. [miniSD](https://huggingface.co/lambdalabs/miniSD-diffusers) is a version of SD that has the same resolution as Modelscope. It's possible to blend scores from the two models together to make the individual frames look better.

### Proposed workflow

I'm open to suggestions, here's an example workflow:
1. Click a checkbox to enable SD blending
2. Set weights for video and image model
3. ???
4. Get video output

**Comment by kabachuha** [comment]
Yes, it kinda is in the plans to make t2v/Deforum integration. We are going to make a some sort of API for the latter to allow other projects interference into its workflow at various generation stages and then it will be much easier
